TumbleBitSetup/Utils.cs

```diff
@@ -55,7 +60,52 @@ internal static int GetOctetLen(int x)
         {
             return (int)Math.Ceiling((1.0 / 8.0) * Math.Log(x+1, 2));
         }
-        
+
+        internal static AsymmetricCipherKeyPair GeneratePrivate(BigInteger exp, int keySize)
+        {
+            SecureRandom random = new SecureRandom();
+            var gen = new RsaKeyPairGenerator();
+            gen.Init(new RsaKeyGenerationParameters(exp, random, keySize, 2)); // See A.15.2 IEEE P1363 v2 D1 for certainty parameter
```

Comment:
@osagga please review this LoC. It looks like this is using a certainty parameter that is too low. Can you look and advise how this LoC should be written.  (I think @NicolasDorier coudl you be using an outdated BC again?)